script.js

```diff
@@ -38,30 +47,122 @@ function shuffle(array) {
 
 let shuffledColors = shuffle(COLORS);
 
-// this function loops over the array of colors
-// it creates a new div and gives it a class with the value of the color
-// it also adds an event listener for a click for each card
 function createDivsForColors(colorArray) {
   for (let color of colorArray) {
-    // create a new div
     const newDiv = document.createElement("div");
 
-    // give it a class attribute for the value we are looping over
     newDiv.classList.add(color);
-
-    // call a function handleCardClick when a div is clicked on
     newDiv.addEventListener("click", handleCardClick);
 
-    // append the div to the element with an id of game
     gameContainer.append(newDiv);
   }
 }
 
-// TODO: Implement this function!
+function resetOldGame() {
```

Comment:
don't we need to reset other values (the ones in the top of this file)?

Reply:
This is embarrassing, but I have a regression here. The 'New Game' button used to clear the entire game board and reset all of the values, but now it's not. I need to look at what I changed. (This is the downside of not using TDD/BDD!) :(

Reply:
I fixed the issue. It was a very simple one. I accidentally deleted part of the id for one of my divs. 

From my testing, it seems that the values are reset during the process  of ending and restarting the game.